pkgs/applications/misc/nerd-font-patcher/default.nix

```diff
@@ -0,0 +1,40 @@
+{ python3Packages, lib, fetchFromGitHub }:
+python3Packages.buildPythonApplication rec {
+  pname = "nerd-font-patcher";
+  version = "2.1.0";
+
+  # The size of the nerd fonts repository is bigger than 2GB, because it
+  # contains a lot of fonts and the patcher.
+  # until https://github.com/ryanoasis/nerd-fonts/issues/484 is not fixed,
+  # we download the patcher from an alternative repository
+  src = fetchFromGitHub {
+    repo = "nerd-fonts-patcher";
+    owner = "betaboon";
+    rev = "180684d7a190f75fd2fea7ca1b26c6540db8d3c0";
+    sha256 = "sha256-FAbdLf0XiUXGltAgmq33Wqv6PFo/5qCv62UxXnj3SgI=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ fontforge ];
+
+  format = "other";
+
+  patchPhase = ''
+    sed -i font-patcher \
+      -e 's,__dir__ + "/src,"'$out'/share/${pname},'
```

Comment:
Substitung pname all over the place makes it harder to understand what is going on because you need to substitute the variable in your head. This is an anti pattern and for many people including me this makes it harder to know what is going on.

Reply:
I understand your point. From my perspective, I had a different feeling while writing the code, since pname is quite long.

Do we have a coding standard, where we can define and discuss such preferences?